mod/thewire/actions/delete.php

```diff
@@ -9,7 +9,7 @@
 
 // Make sure we actually have permission to edit
 $thewire = get_entity($guid);
-if ($thewire->getSubtype() == "thewire" && $thewire->canEdit()) {
+if (elgg_instanceof($thewire, 'object', 'thewire') && $thewire->canEdit()) {
```

Comment:
the check was on subtype which i'm still doing (only better). If it goes to a class check that would be a breaking change and can't go into a patch, this can